modules/testing-3-800.adoc

```diff
@@ -0,0 +1,82 @@
+[[testing-3-800]]
+== Testing 3.8.0 features 
+
+Use the following procedures to test how to enable new features, how to ensure they are working, and the expected results.
+
+=== Enabling and testing the IPv6 and dual-stack protocol family on standalone {productname} deployments 
+
+==== Enabling and testing IPv6 
+
+ IPv6 support is now available with Red Hat Quay 3.8. By enabling IPv6, your {productname} deployment better supports Telco and Edge environments, notably when {productname} serves as a central registry in the form of a hub cluster. 
+ 
+Use the following procedure to enable IPv6 on your standalone {productname} deployment. 
+
+.Prerequisites 
+
+* You have updated {productname} to 3.8.
+
+.Procedure 
+
+. In your deployment's `config.yaml` file, add the `FEATURE_LISTEN_IP_VERSION` parameter and set it to `IPv6`, for example: 
```

Comment:
Maybe add a Prereq that the host and container software platform such as Docker or Podman supports IPv6?